java/ql/test/library-tests/errortype-with-params/Diags.expected

```diff
@@ -0,0 +1 @@
+| Test.java:0:0:0:0 | 1 |
```

Comment:
Slightly confused by this output. Do we have diagnostics with toStrings that are simply "1" or "2"? What do those mean?

Reply:
Appears to come from https://github.com/github/semmle-code/blob/main/extractor-java/src/com/semmle/extractor/java/JavaExtractor.java#L388

I believe @aibaars has a patch coming up to make that more informative.

Reply:
Pulled in https://github.com/github/semmle-code/commit/74c6b3bde70aca02a4355c01cdb4c6e17129be0c to improve this.